codespell_lib/data/dictionary.txt

```diff
@@ -51105,7 +51107,10 @@ speration->separation
 sperations->separations
 sperator->separator
 sperators->separators
+spere->sphere
```

Comment:
Perhaps suggest `spare` and even `spire` and `spore`.

Reply:
I added them but I don't know if it really makes sense because these suggestions don't look as simple typos to me. I mean, one have to be very tired to write `spere` instead of `spore` because the `o` is very far from the `e` on the keyboard :smile: (at least on the keyboard layouts I know)

Reply:
Yes, I wasn't certain about `spire` and `spore`, but `spare` might be the intended word when typing with usual keyboards.

All these suggestions become more probable when typing using a Dvorak layout:

![Dvorak keyboard layout](https://upload.wikimedia.org/wikipedia/commons/thumb/2/25/KB_United_States_Dvorak.svg/800px-KB_United_States_Dvorak.svg.png)

Reply:
Ah indeed, I'm not familiar with Dvorak 😅
Thanks for this suggestion!